### DISCUSSION TITLE
Error 25 while building Carla image in Docker in order to ingest a map created with Road Runner

### DISCUSSION THREAD

**Discussion by ghost** [carla-simulator/carla#4310] [Q&A]
I tried the recommended way to create new map for Carla [https://carla.readthedocs.io/en/0.9.10/tuto_A_add_map/](https://carla.readthedocs.io/en/0.9.10/tuto_A_add_map/). I have the Carla package, so I followed the "Map ingestion in a CARLA package" section. In the very first step I followed the linked instructions to build the image and failed while executing the last one: `docker build -t carla -f Carla.Dockerfile .` .
Copying the text from console, it looked liked this:

\# 4 33154.2 make: *** [package] Error 25
\-\-\-\-\-\-
executor failed running [/bin/sh -c cd /home/ue4 && 
if [ -z ${GIT_BRANCH+x} ]; then git clone --depth 1 https://github.com/carla-simulator/carla.git; 
else git clone --depth 1 --branch $GIT_BRANCH https://github.com/carla-simulator/carla.git; fi && 
cd /home/ue4/carla && 
./Update.sh && 
make CarlaUE4Editor && 
make PythonAPI && 
make build.utils && 
make package && 
rm -r /home/ue4/carla/Dist]: exit code 2

I am using Python 3.7, running Docker on Windows 10 and I have enough disc space to build it. Has anyone encountered the same or similar issue? Does anyone have an idea what the problem is?
Thank you for any advice.

**Comment by e270889o** [comment]
Same problem here. Im using a VM to create it.
Its a clean ubuntu 20.04 vm.
I just did git clone, then 
1. Build the CARLA prerequisites image, with my login and token, it finish sucessful
docker build --build-arg EPIC_USER=<GitHubUserName> --build-arg EPIC_PASS=<GitHubPassword> -t carla-prerequisites -f Prerequisites.Dockerfile .

Then it fails like in the picture in 
2. Build the final CARLA image.
docker build -t carla -f Carla.Dockerfile .

![image](https://user-images.githubusercontent.com/30999190/139193787-4b07585b-c84a-41a8-813e-24d4ee7e6c58.png)

Also, I've tried to navigate to the log file, but i cannot find it in the git clone of carla, or in the docker images
![image](https://user-images.githubusercontent.com/30999190/139195908-262d6adb-d7cf-486d-827a-e6f6293a0dc1.png)

**Reply by Phil095** [reply]
I got the same error when build final carla image: docker build -t carla -f Carla.Dockerfile . . and i cannot find Logs.txt either. Have you solved this issur?

**Reply by e270889o** [reply]
Yes. I increased the swap file. By default it was only 2gb.

With 16gb of ram apparently it fails. So I increased the swap file to 16gb also just in case. 

If you notice, every time you try to run this command, it fails in a different point, so it's not that there is something wrong in the code or a dependencies thing.
